File: Daily Flash/week 4/day 5/MySolutions/C/Program4.c
#include<stdio.h>

void main() {

	printf("\n -------  Pattern   ------ \n\n");
	int ch;
	
	for(int i=0;i<4;i++) {
	
		ch=64;
		for(int j=4;j>i;j--) { 
			printf("    ");
			ch++;	
		}	
		
		for(int k=0;k<=i;k++) {
			printf("%c   ",ch);
			ch++;
		}		
		
		printf("\n\n");
		
	}

}


/*
Output :



(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 5/MySolutions/C$ cc Program4.c
(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 5/MySolutions/C$ ./a.out

 -------  Pattern   ------ 

                D   

            C   D   

        B   C   D   

    A   B   C   D  

*/

File: Daily Flash/week 4/day 5/MySolutions/Python/Program4.py

print("\n  -------- Pattern ---------- \n")

x=4

for i in range(4):
	val=64
	for j in range(x):
		print(end="   ");
		val=val+1
	
	for k in range(0,i+1):
		print(chr(val),end="  ")
		val=val+1
	
	x=x-1
	print("\n")		
	
'''
Output :

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 5/MySolutions/Python$ python3 Program4.py

  -------- Pattern ---------- 

            D  

         C  D  

      B  C  D  

   A  B  C  D  

'''	
